### PR TITLE
Use projectLevelId for retrieving peer reviews

### PR DESCRIPTION
**Pull request by jamescodeorg**
In the curriculum, the code review activities are set up as project template levels with the students writing the code in the first level and then peers reviewing the code in the following level (e.g. CSA unit 3, lesson 3, levels 3+4).  In order to support this, we must store and query for code reviews based on the level id of the project template level.  Support for this was added on the back-end when updating the table (#46528) and model (#46530).  This PR plumbs the value through from the front-end to the API to the back-end.

Note: We decided to keep the actual level id that the code review was opened against even though we are not currently using it.  This information might be useful for debugging or for future iterations where we might show a list of open code reviews and need to know which level to open to show the code review.

## Links

- [tech spec](https://docs.google.com/document/d/1ZtvE5fZPbndZsT3gGLc1ZuH7WcdSRdCj1Tc09EwCQJ4/edit)
- jira ticket: [LP-2380](https://codedotorg.atlassian.net/browse/LP-2380) and subtasks

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/templates/instructions/codeReviewV2/CodeReviewDataApi.js [hunk 1]
```diff
@@ -19,8 +20,8 @@ export default class CodeReviewDataApi {
       url: `/code_reviews/peers_with_open_reviews`,
       type: 'GET',
       data: {
-        levelId: this.levelId,
-        scriptId: this.scriptId
+        scriptId: this.scriptId,
+        projectLevelId: this.projectLevelId
```
Comment (maureensturgeon) [inline apps/src/templates/instructions/codeReviewV2/CodeReviewDataApi.js]: Does this need to be `this.projectLevelId || this.levelId`?

**Review by maureensturgeon — APPROVED** [review]
Thanks so much for taking this on. Were you able to test the changes locally?

File: apps/src/templates/instructions/codeReviewV2/CodeReviewDataApi.js (hunk 1 above)
Comment (jamescodeorg) [reply apps/src/templates/instructions/codeReviewV2/CodeReviewDataApi.js]: Good catch, thanks!

**Comment by jamescodeorg** [comment]
Yes, I was able to test the changes locally.

**Pull request merged**